src/provider/file/mod.rs

```diff
@@ -289,6 +311,12 @@ impl<T: crate::search::Search + Send + Sync> Provider for FileProvider<T> {
     {
         debug!("Creating parcel with SHA {}", parcel_id);
 
+        // TODO: Should we check the bindle ID here or stick with the assumption that most terminal
```

Comment:
@technosophos I'd like your thoughts on this so I can follow up in another PR. I got to thinking while I was working on this and this was the result 😅

Reply:
My got feel is that we want the bindle ID here

Reply:
Ok I’ll do that as a follow up PR to this one to keep it tidy